docs/reference/content/driver-async/getting-started/quick-tour-admin.md

```diff
@@ -223,7 +223,7 @@ For more information about text search see the [text index]({{< docsref "/core/i
 ## Running a command
 
 While not all commands have a specific helper, however you can run any [command]({{< docsref "/reference/command">}})
-by using the [`runCommand()`](http://api.mongodb.org/java/3.1/?com/mongodb/async/client/MongoDatabase.html#runCommand-org.bson.conversions.Bson-com.mongodb.ReadPreference-com.mongodb.async.SingleResultCallback-) 
+by using the [`runCommand()`](http://api.mongodb.org/java/3.2/?com/mongodb/async/client/MongoDatabase.html#runCommand-org.bson.conversions.Bson-com.mongodb.ReadPreference-com.mongodb.async.SingleResultCallback-) 
```

Comment:
Update to use apiref - so we dont have to tweak the version each time. eg: 

    [`runCommand()`]({{< apiref com/mongodb/async/client/MongoDatabase.html#runCommand-org.bson.conversions.Bson-com.mongodb.ReadPreference-com.mongodb.async.SingleResultCallback-" >}})